cluster/addons/rbac/node-proxy-role.yaml

```diff
@@ -10,15 +10,9 @@ rules:
   - ""
   resources:
   - nodes/proxy
-  verbs:
-  - create
-  - get
-- apiGroups:
-  - ""
-  resources:
   - nodes/log
   - nodes/stats
   - nodes/metrics
   - nodes/spec
```

Comment:
I wouldn't expect a role called "node-proxy" to have mutation rights to endpoints other than proxy.